### PR TITLE
Add Del key behavior

### PR DESCRIPTION
**Pull request by mralve**
This just adds the default text editing behavior for the delete key, at least on windows.

### CODE REVIEW

File: src/widget/textbox.rs [hunk 1]
```diff
@@ -460,6 +474,18 @@ fn next_grapheme(src: &str, from: usize) -> usize {
     }
 }
 
+/// Checks if there is a next character from the given index.
+fn next_grapheme_exsists(src: &str, from: usize) -> bool {
```
Comment (zaynetro) [inline src/widget/textbox.rs]: Typo `next_grapheme_exsists` -> `next_grapheme_exists`

Comment (mralve) [reply src/widget/textbox.rs]: Wops thanks for catching that.

**Comment by futurepaul** [comment]
Thanks for this!

I haven't tested on Windows but this works on my Macbook with fn+delete and with my Bluetooth keyboard's del key.

**Comment by raphlinus** [comment]
Thanks for the contribution, and to @futurepaul for the review.

**Pull request merged**